src/Compute/Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -4944,4 +4944,38 @@ function Test-VirtualMachineScaleSetSecurityTypeUpdate
         # Cleanup
         Clean-ResourceGroup $rgname;
     }
-}
\ No newline at end of file
+}
+
+<#
+.SYNOPSIS
+Test Virtual Machine Scale Set defaulting to OrchestrationMode: Flexible when property is not provided.
+#>
+function Test-VirtualMachineScaleSetDefaultToFlexibleOrchestrationMode
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName;
+    $loc = Get-ComputeVMLocation;
+
+    try
+    {
+        # Common
+        New-AzResourceGroup -Name $rgname -Location $loc -Force;
+
+        # New VMSS Parameters
+        $vmssName1 = 'vmss1' + $rgname;
+
+        $vmssConfig = New-AzVmssConfig -Location $loc -UpgradePolicyMode 'Manual' -SinglePlacementGroup $true
```

Comment:
wait, is SinglePlacementGroup a bool or a switchparameter?

Reply:
in New-AzVmssConfig, it is bool for some reason.